logistics/scripts/modules/tools/ToolsHelper.psm1

```diff
@@ -161,7 +161,7 @@ function Test-DotNetCore {
     param (
         [Parameter()]
         [int]
-        $RequiredMajor = 6,
+        $RequiredMajor = 8,
```

Comment:
This function throws an exception when I run it locally, so there's something else that needs to be adjusted.

![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/9324390/845e3acb-536c-4fc0-9969-6f68cfac2209)

Reply:
Sorry, I made a mistake while testing. This is working fine. 
![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/9324390/d060e790-1403-4594-a842-b577b9cf1647)